nuxeo-jsf/nuxeo-platform-ui-web/src/main/resources/tinymce/license.txt

```diff
@@ -1,5 +1,5 @@
-		  GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
The diff shows for this file and many others `100644 → 100755` so they have been made executable, which is actually not needed.